pulp_container/tests/functional/constants.py

```diff
@@ -26,47 +26,18 @@
 CONTAINER_IMAGE_URL = urljoin(PULP_FIXTURES_BASE_URL, "container/busybox:latest.tar")
 """The URL to a Container image as created by ``docker save``."""
 
-# hello-world is the smalest container image available on docker hub 1.84kB
-REPO_UPSTREAM_NAME = "hello-world"
-"""The name of a Container repository.
+# GH Packages registry
+REGISTRY_V2 = "ghcr.io"
+REGISTRY_V2_FEED_URL = "https://ghcr.io"
 
-This repository has several desireable properties:
+# a repository having the size of 1.84kB
+REPO_UPSTREAM_NAME = "pulp/hello-world"
```

Comment:
can you rename this to a more explicit name? pulp_hello_world for example?

Reply:
Did you mean this:
```suggestion
PULP_HELLO_WORLD_REPO = "pulp/hello-world"
```
?